fetch.js

```diff
@@ -459,6 +469,10 @@
         xhr.setRequestHeader(name, value)
       })
 
+      if (request.signal && request.signal.addEventListener) {
```

Comment:
Can we assume that if `request.signal` isn't null, it will always define `addEventListener`?

Reply:
Sure thing :v: